src/Nelmio/Alice/Fixtures/Fixture.php

```diff
@@ -127,11 +127,13 @@ function ($flag) {
             }
         );
 
-        return array_map(
-            function ($extension) {
-                return str_replace('extends ', '', $extension);
-            },
-            $extensions
+        return array_reverse(
```

Comment:
why the `array_reverse`? This will mess up the extending order which can be very problematic